modules/pymol/editing.py

```diff
@@ -24,6 +24,7 @@
           boolean_sc,boolean_dict,safe_list_eval, is_sequence, \
           DEFAULT_ERROR, DEFAULT_SUCCESS, _raising, is_ok, is_error              
     from chempy import cpv
+    import pymol.wizard.dragging
```

Comment:
There's a chance that this now creates a circular import with Python 2 -- @speleo3 is there a CI test suite that can help catch these things?